src/components/sbb-select/sbb-select.tsx

```diff
@@ -0,0 +1,613 @@
+  private _onOptionDeselected(optionSelectionChange: SbbOptionEventData): void {
+    if (this.multiple) {
+      this.value = (this.value as string[]).filter(
+        (el: string) => el !== optionSelectionChange.value
+      );
+
+      this.input.emit();
+      this.change.emit();
+    }
+  }
+
+  private _attachOpenPanelEvents(): void {
+    this._openPanelEventsController = new AbortController();
+
+    // Recalculate the overlay position on scroll and window resize
+    document.addEventListener('scroll', () => this._setOverlayPosition(), {
+      passive: true,
+      signal: this._openPanelEventsController.signal,
+    });
+    window.addEventListener('resize', () => this._setOverlayPosition(), {
+      passive: true,
+      signal: this._openPanelEventsController.signal,
+    });
+
+    window.addEventListener('click', (event) => this._onBackdropClick(event), {
+      signal: this._openPanelEventsController.signal,
+    });
+  }
+
+  private _onKeyDown(event: KeyboardEvent): void {
+    if (this.disabled || this.readonly) {
+      return;
+    }
+
+    if (this._state === 'opened') {
+      this._openPanelKeyboardInteraction(event);
+    }
+    if (this._state === 'closed') {
+      this._closePanelKeyboardInteraction(event);
+    }
+  }
+
+  private _closePanelKeyboardInteraction(event: KeyboardEvent): void {
+    if (this._checkForLetterSelection(event)) {
+      return this._setNextActiveOptionByText(event);
+    }
+
+    switch (event.key) {
+      case 'Enter':
+      case ' ':
+      case 'ArrowDown':
+      case 'ArrowUp':
+        event.preventDefault();
+        this.open();
+        break;
+    }
+  }
+
+  private _openPanelKeyboardInteraction(event: KeyboardEvent): void {
+    if (this.disabled || this.readonly || this._state !== 'opened') {
+      return;
+    }
+
+    if (this._checkForLetterSelection(event)) {
+      return this._setNextActiveOptionByText(event);
+    }
+
+    switch (event.key) {
+      case 'Escape':
+      case 'Tab':
+        this.close();
+        break;
+
+      case 'Enter':
+      case ' ':
+        event.preventDefault();
+        this._selectByKeyboard();
+        break;
+
+      case 'ArrowDown':
+      case 'ArrowUp':
+        event.preventDefault();
+        this._setNextActiveOption(event);
+        break;
+
+      case 'Home':
+      case 'PageUp':
+        event.preventDefault();
+        this._setNextActiveOption(event, 0);
+        break;
+
+      case 'End':
+      case 'PageDown':
+        event.preventDefault();
+        this._setNextActiveOption(event, this._filteredOptions.length - 1);
+        break;
+    }
+  }
+
+  private _checkForLetterSelection(event): boolean {
+    return (
+      event.key === 'Backspace' ||
+      event.key === 'Clear' ||
+      (event.key.length === 1 &&
+        event.key !== ' ' &&
+        !event.altKey &&
+        !event.ctrlKey &&
+        !event.metaKey)
+    );
+  }
+
+  private _setNextActiveOptionByText(event): void {
```

Comment:
It happens because of the  `case ' ':` in the `_openedPanelKeyboardInteraction` method; also, the space is not a 'valid' char in the `_checkForLetterSelection` method. 
Should we fix it? I see that the same problem happens in Material https://stackblitz.com/run?file=src%2Fexample%2Fselect-value-binding-example.html

Reply:
If it's easy solvable, yes, but probably it's not :D and I see that in Material there is the same problem, so maybe we can live with it

Reply:
Did some tests, quite hard to manage :| if it's ok for you, we could leave it as it is

Reply:
ok for me!